src/main/java/de/adesso/service/FileTransfer.java

```diff
@@ -32,37 +23,17 @@ public FileTransfer(ConfigService configService) {
     void copyFile(File source, File dest) {
         try {
             if (source.lastModified() != dest.lastModified()) {
-                LOGGER.info("Copy file from " + source.getAbsolutePath() + " to " + dest.getAbsolutePath());
+                LOGGER.debug("Copy file from " + source.getAbsolutePath() + " to " + dest.getAbsolutePath());
                 if (!dest.exists()) {
                     dest.getParentFile().mkdir();
                 }
                 Files.copy(source.toPath(), dest.toPath(), StandardCopyOption.REPLACE_EXISTING,
                         StandardCopyOption.COPY_ATTRIBUTES);
             }
```

Comment:
Single Level Of Abstraction (SLA) - Zugegeben ist es auf GitHub noch ein bisschen schwieriger zu lesen, weil neue & entferne Codesnippets vermischt sind. Dennoch:

SLA - Verwandt zu SRP. Eine Methode sollte eine Aufgabe erledigen. Die Abstraktion mehrerer Ebenen weist darauf hin, dass die Methode _mehr_ als das tut. Konkret: Geschachtelte Blöcke sind böse & erschweren die Les- & Testbarkeit. 1 Funktion - 1 Methode